MiniAn/minian_main.py

```diff
@@ -183,19 +196,88 @@ def preview(minian_parameters):
     cluster.close()
 
 
+def penalties_preview(minian_params):
+    """
+    Saves penalties_preview
+    """
+    # Start cluster
+    print(mn_defs.Messages.START_CLUSTER, flush=True)
+    cluster = LocalCluster(**minian_params.params_LocalCluster)
+    annt_plugin = TaskAnnotation()
+    cluster.scheduler.add_plugin(annt_plugin)
+    client = Client(cluster)
+
+    # MiniAn CNMF
+    intpath = os.path.join(minian_params.paths[defs.Parameters.Path.TMP_DIR], mn_defs.Folder.INTERMEDIATE)
+    subset = {"frame": slice(*minian_params.preview_params[defs.Parameters.Preview.RANGE])}
+
+    minian_ds  = open_minian(intpath)
+    Y_hw_chk = minian_ds["Y_hw_chk"]
+    Y_fm_chk = minian_ds["Y_fm_chk"]
+
+    file_, chk, _ = load_chunk(intpath, subset, minian_params)
```

Comment:
I guess there should be other way to get the `chk`

Reply:
for example, we don't have to run the whole `load_chunk()`, but just one function from there